internal/provider/networking_resource_test.go

```diff
@@ -315,7 +315,7 @@ resource "cockroach_cluster" "dedicated" {
     cloud_provider = "AWS"
     dedicated = {
         storage_gib = 15
-        machine_type = "m5.large"
+        num_virtual_cpus = 2
```

Comment:
Some of the states like this one needs to be updated as well: https://github.com/cockroachdb/terraform-provider-cockroach/blob/725942c94aa64b9a252a7fb5e69a03cf70c15b42/internal/provider/networking_resource_test.go#L130-L135